src/test/java/io/vertx/mqtt/test/server/MqttServerClientCertSslTest.java

```diff
@@ -0,0 +1,261 @@
+/*
+ * Copyright 2016 Red Hat Inc.
```

Comment:
Having Red Hat is ok. For now let's leave 2016, then I'll change it. No worries ;)